serialization/src/extras/non_empty_vec.rs

```diff
@@ -21,7 +21,7 @@ use serialization_core::{Decode, Encode};
 /// - If the Vec has data, it encodes to just the Vec, the Option is omitted
 /// - If the Vec has no data, it encodes to None
 /// - Some(vec![]) and None are equivalent when encoded, but when decoded result in None
-#[derive(Debug, PartialEq, Eq, Clone, serde::Serialize, serde::Deserialize)]
+#[derive(Debug, PartialEq, Eq, Clone, PartialOrd, Ord, serde::Serialize, serde::Deserialize)]
```

Comment:
I'm not sure this is a necessary addition. I tracked this to the source and these can be removed. Please don't add them unless necessary.

Reply:
It's required because I added `AccountCommand::FillOrder` with `OutputValue`. Comparison is used to store `TxInput` in BTreeSet and check uniqueness of inputs.